### DISCUSSION TITLE
編集リクエストで変更内容入力モーダルを表示するボタンがサブミットされそうで押すのに躊躇してしまう

### DISCUSSION THREAD

**Discussion by ozaki25** [increments/qiita-discussions#939] [要望・アイデア・不具合]
編集リクエストで、修正後に押す「編集リクエストを送信」のボタンが押したらそのまま送信されてしまいそうで躊躇してしまいました
<img width="386" alt="image" src="https://github.com/user-attachments/assets/bdf83856-d2ae-48fd-922d-53cccc8d02db" />

実際にはボタンを押すとモーダルが出て修正内容の説明を入力することができますが、慣れていない人だと説明の入力等なしにいきなり送信されてしまいそうで離脱してしまいそうな気がしました

**Comment by katouriko** [comment]
@ozaki25 
ご意見ありがとうございます🙏

>実際にはボタンを押すとモーダルが出て修正内容の説明を入力することができますが、慣れていない人だと説明の入力等なしにいきなり送信されてしまいそうで離脱してしまいそうな気がしました

こちら参考にさせていただきます🙇‍♂️
着手まで時間がかかる可能性がありますが、検討いたします！

**Comment by makoto-ogata** [comment]
@ozaki25

お待たせいたしました。
ご意見いただきました通り、編集リクエストのエディターの次には、編集リクエストの説明を入力するモーダルが表示されます。

エディターの次のステップがあることを想像しやすいように、エディター右上のボタンのラベルを下記のように変更いたしました。
「編集リクエストを送信」→「送信設定へ」

合わせて、下記のUIの見直しも行っております。
- 編集リクエストの送信設定画面のUIの見直し
- 記事・質問のエディター右上のボタンのラベルの見直し

詳細は下記のリリースノートに記載しております。
[Qiita リリースノート 2025-01-20](https://qiita.com/release-notes/2025-01-20#%E3%82%A8%E3%83%87%E3%82%A3%E3%82%BF%E3%83%BC%E5%8F%B3%E4%B8%8A%E3%81%AE%E3%83%9C%E3%82%BF%E3%83%B3%E3%81%AE%E3%83%A9%E3%83%99%E3%83%AB%E3%81%A8%E7%B7%A8%E9%9B%86%E3%83%AA%E3%82%AF%E3%82%A8%E3%82%B9%E3%83%88%E3%81%AE%E9%80%81%E4%BF%A1%E8%A8%AD%E5%AE%9A%E7%94%BB%E9%9D%A2%E3%82%92%E6%94%B9%E5%96%84%E3%81%97%E3%81%BE%E3%81%97%E3%81%9F)

**Reply by ozaki25** [reply]
分かりやすくていいと思います！ありがとうございます！
